docs/00-manuscripts/ieee/ieee-manuscript.tex

```diff
@@ -433,18 +433,20 @@ \subsubsection{Filter methods}
 The goal is to optimize the number of ranked features to the point at which the model achieves the best performance.
 
 % Ensemble filter methods
-Besides the concept of choosing a specific filter method to rank variables, studies showed that combining several filters using statistical operations such as 'minimum' or 'mean' is able to enhance the predictive performance of the resulting models, especially when applied to multiple datasets \cite{abeel2010, drotar2017a}.
+Besides the concept of choosing a specific filter method to rank variables, studies showed that combining several filters using statistical operations such as 'minimum' or 'mean' can enhance the predictive performance of the resulting models, especially when applied to multiple datasets \cite{abeel2010, drotar2017a}.
 This approach is referred to as 'ensemble filtering' \cite{dietterich2000}.
 Ensemble filters align with the recent rise of the 'ensemble' approach in machine learning which uses the idea of stacking to combine the predictions of multiple models, aiming to enhance predictive performance \cite{polikar2012, feurer2015, bolon-canedo2019}.
+% JM: I have no idea what the Borda ensemble filter is. Maybe add one more sentence on this specific filter
 In this work the 'Borda' ensemble filter was applied \cite{drotar2017a}.
 Its final feature order is determined by the sum of all single filters ranks.
 
 % Ensuring a fair weighting in the ensemble
-Filter methods can be grouped into classes: Correlation-based, entropy-based, linear and non-linear methods.
+% JM: reading further on, it might be that there are only three classes, please refine, Patrick!
```

Comment:
You are right that is confusing. There are three classes and each has one exclusive option in this case. I should make that more clear.